### ISSUE TITLE
Specific timeframe displayed on frontpage

### ISSUE THREAD

**Issue by PeterFleurkens** [Pryx/server-status#29]
It would be nice to have a feature to select a timeframe or number of incidents displayed on the page.
Show only incidents from the past 2 months or 1 year. But still be able to see the incidents in the system but not for the public feed.

**Comment by Pryx** [comment]
This is a good idea. Blocked on #23
